changed scopes to standard to use themes

andreas-hafner opened the pull request:
The scopes in the tmlanguage file for cnc have been changed to standard scopes with additional subranges. Now you can use other themes because the range falls back to the known range. With ISG themes it is more granular than with standard themes. for example GCode and MCode in the ISG themes are distinguished again in the coloring. This does not happen in themes like Dark or Dark+.

This will close #18

lukashettler-isg commented:
Thanks @andreas-hafner 

To get a good highlighting result it's necessary to define syntax groups in the CNC-language. 

I would propose the following groups for syntax highlighting: 

1. G-Functions, M/H-Functions
2. Variables, @P/P-Variables
3. $-Commands
4. L/LL, L/LL CYCLE
5. #-Commands
6. Strings, constants, numbers
7. Comments
8. Block-numbers, program names
9. F/T/S/D-technology data

andreas-hafner commented:
> Thanks @andreas-hafner
> 
> To get a good highlighting result it's necessary to define syntax groups in the CNC-language.
> 
> I would propose the following groups for syntax highlighting:
> 
>     1. G-Functions, M/H-Functions
> 
>     2. Variables, @P/P-Variables
> 
>     3. $-Commands
> 
>     4. L/LL, L/LL CYCLE
> 
>     5. #-Commands
> 
>     6. Strings, constants, numbers
> 
>     7. Comments
> 
>     8. Block-numbers, program names
> 
>     9. F/T/S/D-technology data

I changed the grouping and removed unused scopes from the isg themes and aligned the two theme files

lukashettler-isg reviewed (approved):
Thank you @andreas-hafner 
I will merge your pull request now.
